src/program.js

```diff
@@ -34,7 +34,7 @@ export class Program {
       if (!commandOptions) {
         return;
       }
-      return yargs.options(commandOptions);
+      return yargs.env('WEB_EXT').options(commandOptions);
```

Comment:
I think the `WEB_EXT` string should be referenced from a constant so it can be shared with https://github.com/mozilla/web-ext/blob/master/src/program.js#L119 I typically don't like using constants for strings like this but it would be confusing if someone changed it in one place and not the other. Using a shared constant will prevent that from happening.